README.md

```diff
@@ -1464,6 +1465,16 @@ Type | Interface               | Value type | Description
 out  | evt.notification.report | object     | Notification reports that are sent by devices.
 out  | evt.sensor.report       | object     | Multilevel sensor reports with unknown sensor type or/and unknown unit.
 out  | evt.meter.report        | object     | Meter reports with unknown meter type or/and unit.
+in   | evt.meter.reset         | null       | Resets all historical readings.
```

Comment:
Does this interface reset ALL unknown meters or just one random, or two random, or?

Reply:
From what I see in the documentation it resets all readings from the device